[CS2113-W13-2] FitnessDuke

EangJS opened the pull request:
FitnessDuke is a CLI based workout tracker and guider, that helps individuals get them started on proper workout routines on a fixed schedule in the hopes of getting them fit and healthier.

Ng-YZ commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,230 @@
+<p>
+Figure 1.5
+</p>
+</div>
```
![image](https://user-images.githubusercontent.com/88542282/228424227-58adfe39-9b87-4d5a-a84d-0677016ea451.png)
Perhaps can consider using reference frame over `ExerciseSessionCommandHandler`/`GeneralCommandHandler` then would also be nice to have separate sequence diagram in the next section(s) to show details.

Ng-YZ commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,230 @@
+<p>
+Figure 1.6
+</p>
+</div>
```
Is this figure meant to be the same as the one above? Would be nice if its a diagram showing how `ExerciseSessionCommandHandler` relates to the `ExerciseStateHandler`

Ng-YZ commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,230 @@
+<p>
+Figure 1.7
+</p>
+</div>
```
![image](https://user-images.githubusercontent.com/88542282/228426090-29c483a8-6d7d-40e6-baae-0aefcf46376a.png)
Could consider including the enumeration values for `ErrorMessages`

Ng-YZ replied:
![image](https://user-images.githubusercontent.com/88542282/228426982-b58045e4-3bcc-4575-a81d-4c6d0c1a455e.png)
Not very sure what this diagram is trying to show, composition seems a bit weird?

Ng-YZ commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,230 @@
+| V2.0    | - User                                                                                                   | - be able to start a workout and set it as complete and save it                                                                 | - reflect and track my progress                                                |
+| V2.0    | - User looking for motivation                                                                            | - be able to track my workout history as statistics                                                                             | - better visualise my overall progress                                         |
+| V2.0    | - user with little to no experience with exercise                                                        | - be given instructions for the specific exercise that I am working on                                                          | be educated on how to complete the exercise correctly                          |
+| V2.0    | - User who wants to stay motivated to workout </br> - User who wants to feel good about my past workouts | - See myself be able to accomplish or achieve incrementally greater goals </br> - Keep track of all my exercises                | - Continue to stay motivated in making exercise a fun, long-lasting habit      |
```
![image](https://user-images.githubusercontent.com/88542282/228426549-184740c8-7d53-42ac-a406-1670926d288e.png)
Seems like some slight formatting issue here

EangJS replied:
Yup, thats a known issue.Will correct them in the future iteration(s) 👍

Ng-YZ reviewed (commented):
Easy to read Developer Guide in general, could include slightly more details in some sections though!

chinhan99 commented on docs/UML/Images/Ui.png:
Incorrect syntax such as "C" logo and the circle symbols. Do stick to cs2113 syntax

chinhan99 replied:
Check the arrows as discussed

chinhan99 commented on docs/UML/Images/overall_sequence-0.png:
Do add the termination "cross" if necessary

chinhan99 commented on docs/UML/Images/CommandHandler.png:
Check on the activation bars. and remove the class boxes at the bottom

chinhan99 commented on docs/UML/Images/ErrorMessagesEnum.png:
Maybe can add more description to explain the way the classes interact with each other.

chinhan99 reviewed (commented):
Overall good attempt on your diagrams. Do note the syntax errors as discussed ! Thanks!

MingEn82 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+The class diagram as shown in Figure *1.3* illustrates the structure of the different classes in Ui.
+
+<div align="center">
+<img src="UML/Images/Ui.png"/>
```
![image](https://user-images.githubusercontent.com/89753550/228455660-b077e238-cac9-4d61-9c08-99d7b288bf9b.png)
For the class diagram, do consider removing the "C" logo in front of the class name and change the visibility to +/-/# instead as it is not the intended styling of in this module

MingEn82 commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,239 @@
+
+Accounts for the different scenarios that may trigger an error during user's interactions with the program
+
+#### Error Message Handling
+Enumeration: [```ErrorMessages.java```]
+All error messages are stored in the ErrorMessage enumeration for easy access across different classes that could run into similar exceptions.
+<img src="UML/Images/ErrorMessagesEnum.png"/>
````
![image](https://user-images.githubusercontent.com/89753550/228457666-23967c70-9f4f-46a4-832e-42cc9a25f598.png)
Do consider changing the dotted solid arrow as I'm not sure we have learnt it in this module, hence readers may be confused by what it means

Magmanat replied:
The dotted line with the black diamond is not a valid uml arrow representation which we have learnt in the module, possibly you meant to use a hollow diamond instead to represent aggregation?

![image](https://user-images.githubusercontent.com/74498975/228731371-25dd64c5-0a13-4bd0-ba33-69e4cf98d6a0.png)

Magmanat replied:
The dotted line with the black diamond is not a valid uml arrow representation which we have learnt in the module, possibly you meant to use a hollow diamond instead to represent aggregation?

![image](https://user-images.githubusercontent.com/74498975/228740700-89f2f923-dfb2-4193-9288-7b6c2d848212.png)

MingEn82 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+
+1. Information messages that describe the functionalities of the program (`Greet` , `Bye`, `PrintExercises` classes)
+2. Help messages that details the usage of the possible commands available in the program (`PrintHelpMessage` class)
+3. Error messages that trigger when a user's input is incorrect and provides an explanation to the
```
Maybe can give some examples of how the error messages are triggered

avielcx replied:
It will be good to show examples of errors

MingEn82 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+* Handles when there is no exercise ongoing, granting access to generating new exercises with different confines.
+
+<div align="center">
+<img src="UML/Images/CommandHandler.png"/>
```
![image](https://user-images.githubusercontent.com/89753550/228485331-c124bfaa-7ec2-4395-ab38-9225f7ffd828.png)

Can consider removing the class boxes at the bottom

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+<div align="center">
+<img src="UML/Images/overall_sequence-0.png"/>
+<p>
+Figure 1.2
```
It will be good to show the end of class usage using the cross

avielcx replied:
It will be good to be consistent with the color coding of the classes. commandHandler and Storage has the same colour

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+| V2.0    | - User                                                                                                   | - be able to start a workout and set it as complete and save it                                                                 | - reflect and track my progress                                                |
+| V2.0    | - User looking for motivation                                                                            | - be able to track my workout history as statistics                                                                             | - better visualise my overall progress                                         |
+| V2.0    | - user with little to no experience with exercise                                                        | - be given instructions for the specific exercise that I am working on                                                          | be educated on how to complete the exercise correctly                          |
+| V2.0    | - User who wants to stay motivated to workout </br> - User who wants to feel good about my past workouts | - See myself be able to accomplish or achieve incrementally greater goals </br> - Keep track of all my exercises                | - Continue to stay motivated in making exercise a fun, long-lasting habit      |
```
There is a formatting issue here

avielcx commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,239 @@
+Test cases:  ```o``` , ```hi```
+Expected: Error details will be shown in the terminal
+### ```find``` command
+1. Find a set of exercises based on a specified keyword : ```find [keyword]```
````
It will be good to show an example to the users

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+| V2.0    | - User who wants to stay motivated to workout </br> - User who wants to feel good about my past workouts | - See myself be able to accomplish or achieve incrementally greater goals </br> - Keep track of all my exercises                | - Continue to stay motivated in making exercise a fun, long-lasting habit      |
+
+## Non-Functional Requirements
+1. The program should be able to generate a list of exercises within 5 seconds
```
It will be good to have a better problem recognition. 5 seconds may not be a day to day use of problem recognition

Magmanat commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+* Handles the creation of user data file when previous one is missing or corrupted
+* Handles the loading of user data and plans upon start of the program
+
+The class diagram as shown in *Figure 1.4* illustrates the structure of the different classes in Storage.
+<div align="center">
+<img src="UML/Images/Storage.png"/>
+<p>
+Figure 1.4
+</p>
+</div>
```
Diagram looks neat, however Storage Inherits from two different interfaces, is this possible in Java? 

![image](https://user-images.githubusercontent.com/74498975/228731014-fa137f91-32d7-4650-9ddc-95657b588bb0.png)

Magmanat replied:
The `StorageManager` has an association with `UserPlansStorage` and `UserCareerStorage`, however this is might be already implied and not needed since `StorageManager` implements `Storage` which inherits from `UserPlansStorage` and `UserCareerStorage`.

![image](https://user-images.githubusercontent.com/74498975/228731129-439d027c-03a1-4600-a3ba-b1181ab4a0b0.png)

Magmanat replied:
The `StorageManager` has an association with `UserPlansStorage` and `UserCareerStorage`, however this is might be already implied and not needed since `StorageManager` implements Storage which inherits from `UserPlansStorage` and `UserCareerStorage`.

The `StorageManager` has an association with `UserPlansStorage` and `UserCareerStorage`, however this is might be already implied and not needed since `StorageManager` implements Storage which inherits from `UserPlansStorage` and `UserCareerStorage`.

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+
+{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+### Launch and shutdown of program
```
The launch of the program can be more comprehensive, such as which version of Java, etc.

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+</p>
+</div>
+
+The architecture diagram as shown in Figure 1.1 shows us the high level design of Fitness Duke.
```
It will be good to add more explanations of the architectural components given that the arrows may not be understandable

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+<div align="center">
+<img src="UML/Images/LoadingUserCareerData.png"/>
+<p>
+Figure 1.5
```
There can be more comprehensive explanation to the diagram given the diagram's complexity

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+<div align="center">
+<img src="UML/Images/Storage.png"/>
+<p>
+Figure 1.4
```
There can be more comprehensive explanation to the diagram given the diagram's complexity

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+<div align="center">
+<img src="UML/Images/Ui.png"/>
+<p>
+Figure 1.3
```
There can be more comprehensive explanation to the diagram given the diagram's complexity

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+<div align="center">
+<img src="UML/Images/architecture.png"/>
+<p>
+Figure 1.1
```
Overall comments on the diagrams are that there can be more consistency in the colors across the different diagrams. Some of the diagrams have colours and some don't. It does not seem like it is coded by one person. Also, given that users read the DG for the first time, diagrams, even seemingly simple diagrams can be complicated. It will be good to have more explanations

Magmanat commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+
+**Interaction of components**
+
+<div align="center">
+<img src="UML/Images/overall_sequence-0.png"/>
+<p>
+Figure 1.2
+</p>
+</div>
```
This is a nice diagram which is segregated by colors to represent the classes for easy reference, however this is not consistent with the rest of the developers docs, consider updating the rest of the dev docs to follow this format

Magmanat replied:
This is a nice diagram which is segregated by colors to represent the classes for easy reference, however this is not consistent with the rest of the developers docs, consider updating the rest of the dev docs to follow this format

avielcx commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,239 @@
+Expected: Error details will be shown in the terminal
+### ```find``` command
+1. Find a set of exercises based on a specified keyword : ```find [keyword]```
+2.  Test case: ```find```
````
It will be good to add additional invalid entries of your commands

avielcx commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
-
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+---
+title: Developer Guide
```
I like that the DG is very organized and the Dg is filled with diagram that is adequately understandable. Keep up the good work.

Magmanat commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+
+### Architecture Component
+
+<div align="center">
+<img src="UML/Images/architecture.png"/>
+<p>
+Figure 1.1
+</p>
+</div>
```
Not too sure why there are dotted arrows and solid arrows in the diagram, could be clearer in explaining the difference as developers might not understand the meaning behind them

![image](https://user-images.githubusercontent.com/74498975/228730911-4ee9daa9-64b8-4c93-99fe-74440048976a.png)

Magmanat replied:
Not too sure why there are dotted arrows and solid arrows in the diagram, could be clearer in explaining the difference as developers might not understand the meaning behind them

![image](https://user-images.githubusercontent.com/74498975/228740381-b0b6fc00-f67e-4d0a-9deb-969d006956bc.png)

Magmanat commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,239 @@
+Upon completion of the workout, the user can save their workout if completed, such
+that the completed workout is logged and will be saved for future reference.
+
+To manage this exists the ```ExerciseStateHandler```, which allows for saving, starting
+ending, cancelling workouts.
+<div align="center">
+<img src="UML/Images/CommandHandler.png"/>
+<p>
+Figure 1.7
+</p>
+</div>
````
Possibly missing out on the activation bar for all the classes that are involved in this sequential diagram.

![image](https://user-images.githubusercontent.com/74498975/228731248-def610ea-a2d7-4e7f-a8b7-2c831aa8f8f5.png)

Magmanat replied:
Possibly missing out on the activation bar for all the classes that are involved in this sequential diagram.

![image](https://user-images.githubusercontent.com/74498975/228740635-b818d615-7f81-42cf-80c9-6ea932cf0235.png)

Magmanat reviewed (approved):
Overall Neat Developer Documentation, however could have better clarity on some of the diagrams

Magmanat commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,239 @@
+* Handles the creation of user data file when previous one is missing or corrupted
+* Handles the loading of user data and plans upon start of the program
+
+The class diagram as shown in *Figure 1.4* illustrates the structure of the different classes in Storage.
+<div align="center">
+<img src="UML/Images/Storage.png"/>
+<p>
+Figure 1.4
+</p>
+</div>
```
Diagram looks neat, however Storage Inherits from two different interfaces, is this possible in Java?
![image](https://user-images.githubusercontent.com/74498975/228740513-78662792-a4fc-46d4-86fc-1f4758ed08cb.png)

EangJS replied:
Storage interface extends UserPlansStorsge and UserCareerStorage interfaces. This is similar to AB3.

Magmanat reviewed (commented):
Overall Neat Developer Documentation, however could have better clarity on some of the diagrams

lipkuangchng committed:
improved ui for history command

lipkuangchng committed:
Fixed broken hyperlinks in UG

lipkuangchng committed:
Updated UserGuide to flow better. Also fixed broken hyperlinks.

lipkuangchng committed:
Fixed junit test error

lipkuangchng committed:
fix runtest.bat

EangJS committed:
Merge pull request #174 from L-K-Chng/branch-updateV2Error

Branch update v2 error

lipkuangchng committed:
Merge branch 'master' into branch-updateV2Error

Khulon committed:
minor update to UG

EangJS committed:
Add EangJS @author

lipkuangchng committed:
Merge pull request #175 from Khulon/master

minor update to UG

EangJS committed:
Update EangJS PPP

EangJS committed:
Fix null input

EangJS committed:
Merge branch 'master' of github.com:EangJS/tp

ghzr0 committed:
IPPT calc feature

ChubbsBunns committed:
Merge pull request #176 from EangJS/master

Add PPP and bug fixes

ChubbsBunns committed:
Test TOC

ChubbsBunns committed:
Edit Hashes

lipkuangchng committed:
Merge branch 'master' into branch-updateV2Error

lipkuangchng committed:
Updated User help message

ghzr0 committed:
IPPT calc feature fix

ghzr0 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W13-2/tp

ghzr0 committed:
Checkstyle issues fixed

ghzr0 committed:
Fix gradle issues

ghzr0 committed:
no message

ghzr0 committed:
IO redirection test

ghzr0 committed:
Fix IO redirection windows issue

ghzr0 committed:
no message

ghzr0 committed:
no message

Khulon committed:
added PPP

lipkuangchng committed:
Merge pull request #179 from Khulon/master

added PPP

lipkuangchng committed:
Added details to PPP

lipkuangchng committed:
Merge branch 'master' into branch-updateV2Error

EangJS committed:
Update eangjs.md

ghzr0 committed:
Fixed changes based on suggestions

ghzr0 committed:
Removed redundant methods

lipkuangchng committed:
Added details to PPP

lipkuangchng committed:
fixed runtest.bat

EangJS committed:
Merge pull request #180 from L-K-Chng/branch-updateV2Error

Branch update v2 error

EangJS committed:
Update Storage UML

EangJS committed:
Remove comma from description printing

ghzr0 committed:
1.Updated UG
2. Added individual PPP
3. FIx 'find' feature
4. Replace unknown characters in data.json
5. Added test case for find command

EangJS committed:
Merge pull request #177 from ghzr0/master

IPPT calc feature part 1

Khulon committed:
added new exception clarifying the error that the reason for inability to generate exercises when there are no filters and over 873 is not due to excessive filters, but due to the exercises library limit.

Khulon committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

EangJS committed:
Merge pull request #181 from Khulon/master

EangJS committed:
Remove redundant files

EangJS committed:
Fix merge conflicts

EangJS committed:
Fix find command

lipkuangchng committed:
Search within completed list

EangJS committed:
Update ui tests

EangJS committed:
Remove garbage files

EangJS committed:
Remove additional garbage files

EangJS committed:
Create EXPECTED-UNIX.TXT

EangJS committed:
Edit ui test

lipkuangchng committed:
fixed printexercisedatatest

EangJS committed:
Change 3/4 format

lipkuangchng committed:
fix runtest.bat

lipkuangchng committed:
improved data command

lipkuangchng committed:
minor change

EangJS committed:
ui test

EangJS committed:
Format 3/4 notation

EangJS committed:
Format 3/4 notation

lipkuangchng committed:
checkstyle fixes

lipkuangchng committed:
more checkstyle fixes

lipkuangchng committed:
fixed test

ghzr0 committed:
Merge pull request #182 from EangJS/master

Remove Logger and Bug Fixes

ghzr0 committed:
Remodify IPPT session feature

ghzr0 committed:
Fixed checkstyle errors

ghzr0 committed:
Fix checkstyle error

ghzr0 committed:
Fixed testPrintHelp with updated help message

ghzr0 committed:
no message

ghzr0 committed:
Update tests

ghzr0 committed:
Fix code style for non windows OS

ghzr0 committed:
no message

ghzr0 committed:
Fix EXPECTED.TXT

ghzr0 committed:
Commit SUmmary:
1. Included exceptions and restrictions about inputs for IPPT scoring
2. Include IPPT exercises as part of an exercise session and ability to add into UserCareerData(re-modified)
3.Remodify IPPTCmd to handle user input to start IPPT exercise session

ghzr0 committed:
no message

ghzr0 committed:
Merge pull request #184 from ghzr0/master

Remodify IPPT session feature

ChubbsBunns committed:
Add example output for UG, add clear achievements

ChubbsBunns committed:
Merge remote-tracking branch 'test/master'

ChubbsBunns committed:
Updated UG

ChubbsBunns committed:
Pass help function checks

ChubbsBunns committed:
Pass linux system tests

ChubbsBunns committed:
Remove stray * in EXPECTED.txt

ChubbsBunns committed:
Edit expected.txt file

lipkuangchng committed:
resolved merge conflict

lipkuangchng committed:
fixed merge conflict

lipkuangchng committed:
fixed runtest.bat

EangJS committed:
Merge pull request #183 from L-K-Chng/branch-searchCompletedExercisesFeature

ChubbsBunns committed:
Merge https://github.com/AY2223S2-CS2113-W13-2/tp into documentation-DG-Dylan

ChubbsBunns committed:
Pass checkstyle checks

ChubbsBunns committed:
Merge pull request #186 from ChubbsBunns/documentation-DG-Dylan

Enhance UG

ghzr0 committed:
Update PPP and UG for IPPT calculator feature

EangJS committed:
Merge pull request #189 from ghzr0/master

Update PPP and UG for IPPT calculator feature

EangJS committed:
Update author tags and improve exception handling

EangJS committed:
Minor bug fixes

Improve organisation of command summary
Better elaboration of ippt feature in UG
Ui bug fix for printing empty find command
Alerts for corrupted/missing data

EangJS committed:
Catch keyboard interrupts

EangJS committed:
Update and colourise uml

EangJS committed:
Add V2.1 user stories

EangJS committed:
Make UG TOC autogen

lipkuangchng committed:
Merge pull request #191 from EangJS/master

Minor Fixes

lipkuangchng committed:
added assertions

lipkuangchng committed:
Assertions

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

ChubbsBunns committed:
Fix achievements, update Dylan PPP and UG

ChubbsBunns committed:
Remove default achievementList file

ChubbsBunns committed:
Delete original achievements file

ChubbsBunns committed:
Merge pull request #192 from ChubbsBunns/documentation-DG-Dylan

Fix Achievements, update PPP, help message and UG

lipkuangchng committed:
Updated architecture diagram

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

EangJS committed:
Update UML and DG

EangJS committed:
Fix html tagging

EangJS committed:
Fix extra '-' in table

EangJS committed:
Add UML for saving of user plan

EangJS committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

EangJS committed:
Merge pull request #193 from EangJS/master

lipkuangchng committed:
minor change

ChubbsBunns committed:
Style UG

EangJS committed:
Merge pull request #194 from ChubbsBunns/master

lipkuangchng committed:
improved architecture diagram for dg

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

lipkuangchng committed:
improved data command to show the total number of exercises the user has completed.

ChubbsBunns committed:
Add test for exercise permutations

lipkuangchng committed:
minor change

ChubbsBunns committed:
Fix achievements, add achievement tests

ChubbsBunns committed:
Pass checkstyle

lipkuangchng committed:
updated data command and updated junit test

lipkuangchng committed:
fix runtest.bat

lipkuangchng committed:
Merge pull request #195 from ChubbsBunns/master

Fix achievements, add achievement tests

lipkuangchng committed:
resolve merge conflict

EangJS committed:
Merge pull request #196 from L-K-Chng/branch-updateV2.1

lipkuangchng committed:
Added test for completed exercises search

lipkuangchng committed:
Added junit test for search function

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

lipkuangchng committed:
test for user career data

lipkuangchng committed:
modified test

ChubbsBunns committed:
Modularize achievement test

ChubbsBunns committed:
Refactor test code

ghzr0 committed:
Update DG and create class for testing

ChubbsBunns committed:
Pass checkstyle

ChubbsBunns committed:
Pass runtest.bat check

lipkuangchng committed:
Merge pull request #213 from ChubbsBunns/master

Refactor Test Code, edit difficulty text output and refactor UG

lipkuangchng committed:
modified ug

lipkuangchng committed:
merge conflict resolve

lipkuangchng committed:
fix

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

ghzr0 committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

lipkuangchng committed:
updated UG

lipkuangchng committed:
added tests for user exercise data

lipkuangchng committed:
fix checkstyle

EangJS committed:
Merge pull request #214 from L-K-Chng/branch-updateV2.1

lipkuangchng committed:
added new error message for delete session

Khulon committed:
added new exceptions to quick start command and added tests for help command and quick start command

ghzr0 committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

Khulon committed:
Merge branch 'master' into master

Khulon committed:
fixed checkstyle

Khulon committed:
added tests for plannermode and workout mode

Khulon committed:
fixed bugs

Khulon committed:
fixed bugs

Khulon committed:
fixed bugs

ghzr0 committed:
Merge pull request #215 from Khulon/master

updates to testing

ghzr0 committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

ghzr0 committed:
Commit Summary:
1. Update DG commands test cases
2. Update UG for details of IPPT calculator feature
3. Add comments for methods used in IPPT calculator feature
4. Fix output message for find command
5. Update exceptions and error message details
6. Fix checkstyle errors

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

ghzr0 committed:
1. Update UG to include justifications for max rep limits
2.Update IPPTCalc and IPPTCmd to handle invalid test cases
3.Update Help Message to facilitate valid inputs from user

ghzr0 committed:
Fix gradle test errors

ghzr0 committed:
Fix runtest.sh indentation

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
Delete input.txt

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
no message

ghzr0 committed:
changed runtest.sh file

ghzr0 committed:
no message

ghzr0 committed:
Update Ui test

ghzr0 committed:
Add new expected

ChubbsBunns committed:
Merge pull request #216 from ghzr0/master

Finish group deliverables

lipkuangchng committed:
achievement

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

Khulon committed:
changed description of "static"

Khulon committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

EangJS committed:
Merge pull request #217 from Khulon/master

changed description of "static"

lipkuangchng committed:
Updated PPP, UG and DG. Updated FAQ for UG. Updated CommandList UML and Command column for DG.

lipkuangchng committed:
resolve merge conflict

lipkuangchng committed:
fix checkstyle

EangJS committed:
Merge pull request #218 from L-K-Chng/branch-updateV2.1

Branch update v2.1

lipkuangchng committed:
corrected Ui class diagram

ChubbsBunns committed:
Add DG material

ChubbsBunns committed:
Merge https://github.com/AY2223S2-CS2113-W13-2/tp

ChubbsBunns committed:
Pull other's work

Khulon committed:
added commands to command summary of UG

Khulon committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	plansData.json

Khulon committed:
Merge remote-tracking branch 'origin/master'

lipkuangchng committed:
updated Ui uml class diagram

lipkuangchng committed:
changed emoji to prevent possibility of bug

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

lipkuangchng committed:
Merge pull request #219 from Khulon/master

added commands to command summary of UG

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

lipkuangchng committed:
Added on to the Command Summary of the UG

lipkuangchng committed:
changes to sequence of Command Summary

EangJS committed:
Merge pull request #220 from L-K-Chng/branch-updateV2.1

Branch update v2.1

lipkuangchng committed:
minor changes

ChubbsBunns committed:
Merge pull request #221 from L-K-Chng/branch-updateV2.1

minor changes

EangJS committed:
Add page breaks

EangJS committed:
Add page breaks

ChubbsBunns committed:
Update DG diagrams

EangJS committed:
Update styling

EangJS committed:
Merge pull request #222 from EangJS/master

Minor edits

lipkuangchng committed:
ppp minor change

EangJS committed:
Merge pull request #223 from L-K-Chng/branch-updateV2.1

ChubbsBunns committed:
Update DG

ChubbsBunns committed:
Merge https://github.com/AY2223S2-CS2113-W13-2/tp into documentation-DG-Dylan

ChubbsBunns committed:
Remove unnecessary data files

lipkuangchng committed:
Merge pull request #224 from ChubbsBunns/master

Update DG

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

EangJS committed:
Clean up PPP

EangJS committed:
Merge branch 'master' of github.com:EangJS/tp

EangJS committed:
Clean up PPP

Khulon committed:
edited PPP

Khulon committed:
Merge branch 'AY2223S2-CS2113-W13-2:master' into master

lipkuangchng committed:
add details for ppp

lipkuangchng committed:
Merge pull request #225 from EangJS/master

Finalise PPP

lipkuangchng committed:
Merge pull request #226 from Khulon/master

updated own ppp

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

lipkuangchng committed:
Merge pull request #227 from L-K-Chng/branch-updateV2.1

Finalise PPP

ChubbsBunns committed:
Update PPP and DG

ChubbsBunns committed:
Update UG and DG and PPP

ChubbsBunns committed:
Edit UML diagrams

EangJS committed:
Merge pull request #228 from ChubbsBunns/master

ChubbsBunns committed:
Fix UG

ChubbsBunns committed:
Fix DG

ChubbsBunns committed:
Delete Achievement text

ChubbsBunns committed:
Merge pull request #229 from ChubbsBunns/master

Fix DG

EangJS committed:
Update Code theme

EangJS committed:
Update default.html

EangJS committed:
Merge pull request #230 from AY2223S2-CS2113-W13-2/Styling

Update default.html

ChubbsBunns committed:
Update UG and PPP

ChubbsBunns committed:
Update UG

lipkuangchng committed:
updated UG

lipkuangchng committed:
Merge branch 'master' into branch-updateV2.1

ChubbsBunns committed:
Update UG Cat exercise

ChubbsBunns committed:
Merge pull request #231 from ChubbsBunns/master

Update UG and PPP

lipkuangchng committed:
minor changes

lipkuangchng committed:
Merge pull request #232 from L-K-Chng/branch-updateV2.1

Branch update v2.1

EangJS committed:
add [] to help message
